src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java

```diff
@@ -149,7 +147,7 @@ private static void resolveSSRCAsEndpoint(
      * The <tt>DominantSpeakerIdentification</tt> instance which
      * detects/identifies the active/dominant speaker in {@link #conference}.
      */
-    private final DominantSpeakerIdentification dominantSpeakerIdentification
+    private DominantSpeakerIdentification dominantSpeakerIdentification
```

Comment:
not opposing, just curious why?

Reply:
I must have missed something while rebasing. I want to set it to null in `expire`.